piero-vic/blog-app
# feat: Validations

# feat: Validations

- Add validations.
- Add unit specs for models.
- Add flash messages.
- Solve N+1 problem when fetching all posts and their comments for a user.

NOTE: This project uses Tailwind. Please make sure to run `./bin/dev` before `rails s`. This is needed for the CSS to compile.
